tests.py

```diff
@@ -1815,3 +1815,21 @@ def test_missing_source_files(self):
                 "some files listed as being under source control are missing:\n"
                 "  missing.py",
                 sys.stderr.getvalue())
+
+    def test_broken_symbolic_link(self):
+        from check_manifest import check_manifest
+        self._create_repo_with_code()
+        os.symlink(os.path.join(self.tmpdir, "some-file"),
+                   os.path.join(self.tmpdir, "some-symbolic-link"))
```

Comment:
Seeing the Appveyor CI failure on Windows, I think

```suggestion
        os.symlink(os.path.join(self.tmpdir, "some-file"), "some-symbolic-link")
```

might work better ...